draft-irtf-cfrg-hpke.md

```diff
@@ -375,12 +379,9 @@ context. The key schedule inputs are as follows:
 * `pkSm` - The sender's encoded public key (optional; default
   value `zero(Npk)`)
 
-Senders and receivers MUST validate public keys for correctness.
-For example, when using a DH-based KEM, the sender should check
-that the receiver's key `pkR` is valid, i.e., a point on the
```

Comment:
We lost this recommendation (check `pkR`) in the new text. Can we bring it back?

Reply:
Please let me know if you think this is addressed by my most recent change.